composer.json

```diff
@@ -3,14 +3,14 @@
     "description": "Wrapper for League of Legends API",
     "require": {
         "guzzlehttp/guzzle": "^6.0",
-        "doctrine/cache": "^1.6",
-        "predis/predis": "~1.0"
+        "symfony/cache": "^3.2"
     },
     "keywords": [
         "lol", "api" , "league of legends", "lol api", "library"
     ],
     "require-dev": {
-        "phpunit/phpunit": "^5.2"
+        "phpunit/phpunit": "^5.2",
+        "predis/predis": "~1.0"
```

Comment:
because you put it in the require section at the first place, which is way worse since you force ppl to choose redis as a cache implementation. require dev predis in case of tests requires predis implementation. But if you want to change it on suggest, you can update my PR.